Other/ValetDefines.h

```diff
@@ -43,3 +43,6 @@
 
 /// Compile flag for building against iOS 9.
 #define VAL_IOS_9_OR_LATER (TARGET_OS_IPHONE && __IPHONE_9_0)
+
+/// Error returned from Security API when the application is not entitled to perform the requested action.
+OSStatus const errSecMissingEntitlement = -34018;
```

Comment:
Unfortunately this error is not documented in the API (though a quick search online shows that the mapping is correct). Not sure if this is the best place to dump this definition.